wasmaudioworklet/audioworkletpolyfill.js

```diff
@@ -114,3 +123,6 @@ if(typeof AudioWorkletNode !== 'function') {
         window.audioWorkletProcessors[name] = new processorClass();
     }
 }
+
+// For TypeScript to recognize the file as a "module".
+export {}
```

Comment:
Due to legacy TypeScript functionality from before ES Modules were a thing, any file without an `import` or `export` statement is not considered a module. If the file is not considered a module, the `app.js` file is not able to import it (type error on that import line). Adding the no-op `export {}` line marks the file as a module in TS eyes.